### DISCUSSION TITLE
新增了「優秀專案」分類，歡迎大家分享優秀的Github專案

### DISCUSSION THREAD

**Discussion by chia-kai-chang** [1101-GS3073-C/Python-Community-GS3073-C#290] [Announcements]
之前同學分享了很多優秀的Github專案，如果大家也可以分享，一定可以增廣見聞，未來開發程式事半功倍。

因此新增了「優秀專案」的分類，觀迎大家分享不錯的專案喔~

![image](https://user-images.githubusercontent.com/1768891/159221499-9f3717c8-1a0e-48eb-8a8e-f1fb8ac22f1a.png)

**Comment by Bill671** [comment]
Yolov5 是一個針對物件做辨識的套件，透過內建檔案detect.py，將影像放置默認的data/image裡面，即會輸出辨識後的結果到run/exp{ } ，{ } 為你使用第幾次detect.py，預設有一個exp，之後再輸出一次(再執行一次detect.py)則增加一個exp2，再輸出一次則再增加exp3，以此類推，如果前面輸出不需要，可自行刪除。
模型用作辨識物件，何謂物件，即貓、狗、牛、人類等等甚至是領帶、車子，輸出資料夾會顯示程式認為其為該物件的可能率。 [模型我放在這邊](https://github.com/ultralytics/yolov5)，有需要請自行取用

**Reply by chia-kai-chang** [reply]
@Bill671 這樣的內容可以開一個新討論喔!

**Comment by Bill671** [comment]
Hacktrick 是講述白帽駭客的所有技巧，其中也涵蓋cybersecurity的一些技術，裡面內容其實很深，推薦有一定Python基礎的人詳細閱讀，由於其水非常深，還請小心服用 [連結請點我](https://github.com/carlospolop/hacktricks)

**Reply by chia-kai-chang** [reply]
@Bill671 這樣的內容可以開一個新討論喔!

**Reply by Bill671** [reply]
！…阿阿我誤解老師您的意思了
我以為是所有專案都留言在這下方🤣 好，我之後會留意
